ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestinterfaces.py

```diff
@@ -60,10 +60,9 @@ def generate_test_details(interface: dict, description_template: str) -> tuple(s
             """
             if "Loopback" in description_template:
                 return "up", "up", description_template
-            elif interface["shutdown"]:
+            if interface["shutdown"]:
                 return "adminDown", "down", description_template.format(state="adminDown")
-            else:
-                return "up", "up", description_template.format(state="up")
+            return "up", "up", description_template.format(state="up")
 
         interface_types = [
```

Comment:
Maybe this should be a class variable for this one

Reply:
Agreed. Fixed in 22836aff3e4102ab512a54b400ec10a4213cff27